scripts/cluster_settings

```diff
@@ -1,12 +1,11 @@
 . "${SCRIPTS_DIR}"/lib/source_only
 
 # We need a minimal setup to verify that lighthouse works
-clusters=('cluster1' 'cluster2' 'cluster3')
+clusters=('cluster1' 'cluster2')
 cluster_nodes['cluster1']="control-plane worker worker"
 cluster_nodes['cluster2']="control-plane worker worker"
-cluster_nodes['cluster3']="control-plane worker"
 
-cluster_cni=( ['cluster1']="weave" ['cluster2']="weave" ['cluster3']="weave" )
+cluster_cni=( ['cluster1']="weave" ['cluster2']="weave" )
```

Comment:
we need the cluster3 for round robin and healthcheck test cases.

Reply:
No plan to merge this as lighthouse doesn't care about CNI. This is more of a troubleshooting PR to try and see why it is not working. Wanted to check if using 2 clusters instead of 3 works.

Reply:
Whall we avoid using OVN altogether here?, it should be independent.